<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="edit_btn">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>141</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Редактировать</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="table">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>311</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="words_btn">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>50</y>
     <width>141</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Слова</string>
   </property>
  </widget>
  <widget class="QPushButton" name="coll_btn">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>80</y>
     <width>141</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Сочетания</string>
   </property>
  </widget>
  <widget class="QPushButton" name="conn_btn">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>110</y>
     <width>141</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Связи</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
